test/adaptive_load/utility.cc

```diff
@@ -0,0 +1,45 @@
+#include "test/adaptive_load/utility.h"
+
+namespace Nighthawk {
+
+namespace {
+
+void SetCounterValue(nighthawk::client::Counter* counter, const std::string& name, int value) {
+  counter->set_name(name);
+  counter->set_value(value);
+}
+
+void SetStatisticValues(nighthawk::client::Statistic* statistic, const std::string& id, long min_ns,
+                        long mean_ns, long max_ns, long pstdev_ns) {
+  const long kOneBillion = 1000 * 1000 * 1000;
+  statistic->set_id(id);
+  statistic->mutable_min()->set_seconds(min_ns / kOneBillion);
+  statistic->mutable_min()->set_nanos(min_ns % kOneBillion);
+  statistic->mutable_mean()->set_seconds(mean_ns / kOneBillion);
+  statistic->mutable_mean()->set_nanos(mean_ns % kOneBillion);
+  statistic->mutable_max()->set_seconds(max_ns / kOneBillion);
+  statistic->mutable_max()->set_nanos(max_ns % kOneBillion);
+  statistic->mutable_pstdev()->set_seconds(pstdev_ns / kOneBillion);
+  statistic->mutable_pstdev()->set_nanos(pstdev_ns % kOneBillion);
+}
+
+} // namespace
+
+nighthawk::client::Output MakeStandardNighthawkOutput() {
```

Comment:
possible overkill, but this could take a (deterministic) random generator to add the potential to test with some more entropy. entirely optional, just floating an idea.

Reply:
The random generator might be overkill for now, but I did end up expanding this a bit.